doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -307,23 +307,23 @@ cells:
     
     By default, ``read_excel`` will search the position of the first cell containing the special character ``\`` in the header line in order to determine the number of axes of the array to read. The special character ``\`` is used to separate the name of the two last axes. If there is no special character ``\``, ``read_excel`` will consider that the array to read has only one dimension. For an array stored as:
     
-    | city    | gender\time | 2010 | 2011 | 2012 |
-    | ------- | ----------- | ---- | ---- | ---- |
-    | Brussel | female      | 0    | 1    | 2    |
-    | Brussel | male        | 3    | 5    | 6    |
-    | Paris   | female      | 7    | 8    | 9    |
-    | Paris   | male        | 10   | 11   | 12   |
+    | city     | gender\time | 2010 | 2011 | 2012 |
+    | -------- | ----------- | ---- | ---- | ---- |
+    | Brussels | female      | 0    | 1    | 2    |
```

Comment:
Use eurostat data here too?

Reply:
Yes, it's in the pipe. Give me some time.